devices/A4988/samples/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System.Reflection;
```

Comment:
please copy/paste and adjust one of the other sample file

Reply:
done